src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -260,6 +275,32 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructors(
         return email;
     }
     
+    private List<EmailWrapper> generateFeedbackSessionClosedEmail(
+            CourseAttributes course, FeedbackSessionAttributes session,
+            List<InstructorAttributes> instructors, String template, String subject) {
```

Comment:
You don't need the parameters `template` and `subject` here since they are fixed.